tests/unit/hive_metastore/test_locations.py

```diff
@@ -306,9 +307,9 @@ def test_match_table_external_locations():
 def test_mount_listing_multiple_folders():
     client = create_autospec(WorkspaceClient)
 
-    first_folder = FileInfo("dbfs:/mnt/test_mount/table1/", "table1/", "", "")
-    second_folder = FileInfo("dbfs:/mnt/test_mount/table2/", "table2/", "", "")
-    folder_table1 = FileInfo("dbfs:/mnt/test_mount/table1/_delta_log/", "_delta_log/", "", "")
+    first_folder = FileInfo("dbfs:/mnt/test_mount/table1/", "table1/", 0, "")
```

Comment:
Should have been picked up by the linter, could you trigger the linter for this tests and the tests below by adding a type hint:

```
def test_mount_listing_multiple_folders() -> None
```

Reply:
I'm not understanding, test does have a type hint (return `None`)?

Reply:
I do not see a type hint for `test_mount_listing_multiple_folders`? 

If a method does not have a type hint in its signature then the body will not be linted

Reply:
@JCZuurmond - Not sure I follow. First, there are no methods here, only functions. Second, this does have a return type hint of `None` can you please be clear about what you are looking for?

https://github.com/databrickslabs/ucx/blob/9c8b22be4dd29ecba8ebb9c5ebcf9061b076cd79/tests/unit/hive_metastore/test_locations.py#L307

Reply:
Did you push your changes?

![Screenshot 2024-10-07 at 09 24 24](https://github.com/user-attachments/assets/9dca056a-c9f3-429c-bfcd-4bbaaac08a9f)

Reply:
I probably made the mistake of looking at the comment in context, thus missing that commit. Sorry for the confusion